packages/connector-local/src/connector.ts

```diff
@@ -242,6 +245,17 @@ export default class LocalConnector implements IConnector {
         });
     }
 
+    private watchForWindow() {
```

Comment:
This will work if there is more than one html file?

Reply:
This will only populate the connector for the target HTML file as the connector APIs are tied to that singleton as far as I could tell.

Reply:
Yep, the problem will be with a glob pattern.

I think you need to touch a little bit how the connector works to support multiple HTMLs.

Right now, the fetch is in parallel, but maybe you can refactor the code to emit the `fetch::end::html` serially so you can set the properties in the connector for each html, this way, the connector will support the `parse-html` for all the HTML files. Do you know what I mean?

The idea is when the connector store all the HTMLs in an array, without emits the `fetch::end::html`. Then, iterate that array serially doing:
1. emit the `fetch::end::html`
1. in the event `parse::html` set `this._window`
1. wait until the analysis of the html finish.
1. clean `this._window`
1. go to the next html.

Reply:
Makes sense and doesn't look too difficult from glancing at the code. I'll see if I can incorporate this tomorrow.